vocabulary/All the most important terms from A-Z

```diff
@@ -0,0 +1,568 @@
+# All the most important terms from A-Z
```

Comment:
This file has no extension (should be a MD file) it's also bad practice to have spaces in filenames.